src/main/java/at/ac/tuwien/kr/alpha/grounder/structure/AnalyzeUnjustified.java

```diff
@@ -2,7 +2,10 @@
 
 import at.ac.tuwien.kr.alpha.common.*;
 import at.ac.tuwien.kr.alpha.common.atoms.*;
-import at.ac.tuwien.kr.alpha.grounder.*;
+import at.ac.tuwien.kr.alpha.grounder.Instance;
+import at.ac.tuwien.kr.alpha.grounder.NonGroundRule;
+import at.ac.tuwien.kr.alpha.grounder.Unification;
+import at.ac.tuwien.kr.alpha.grounder.Unifier;
```

Comment:
OK.

Reply:
By the way, this behaviour is controlled in IntelliJ IDEA by the setting `File > Settings > Project Settings > Code Style > Java > Imports > General > Class count to use import with '*'`. Which setting do you use? See also #210

Reply:
The right way to configure this is by modifying `config/checkstyle.xml` which should drive your IDE settings. Configured correctly, this eliminates the question "Which setting do you use?" since it is checked in to the repo and can be changed by PR.

See for example the check [`AvoidStarImport`](https://checkstyle.sourceforge.io/config_imports.html#AvoidStarImport).